components/layouts/Framework/Framework.test.tsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react'
+import '@testing-library/jest-dom/extend-expect'
+import { render, RenderResult } from '@testing-library/react'
+import fetch from 'jest-fetch-mock'
+
+import { LayoutFramework } from '.'
+
+describe('LayoutFramework', () => {
+  let wrapper: RenderResult
+
+  it('fetches and displays the RNDS latest tag', async () => {
+    fetch.mockResponseOnce(JSON.stringify({ tag_name: '1.2.3' }))
+    wrapper = render(<LayoutFramework>Hello, World!</LayoutFramework>)
+
+    expect(fetch).toHaveBeenCalledWith(
+      'https://api.github.com/repos/Royal-Navy/design-system/releases/latest'
+    )
+
+    expect(fetch).toHaveBeenCalledTimes(1)
+
+    expect(await wrapper.findByText('v1.2.3')).toBeInTheDocument()
```

Comment:
Do we need this `await`? If the result is async then the preferred route could be `waitFor`

Reply:
If I use `waitFor` here instead of a simple await on the `findBy` it throws some funny warnings:

<img width="1310" alt="Screenshot 2021-04-30 at 12 13 10" src="https://user-images.githubusercontent.com/48086589/116687734-8cb46100-a9ad-11eb-8f35-352af33f7d64.png">

Reply:
This works..

```
    return waitFor(() => {
      expect(wrapper.queryByText('v1.2.3')).toBeInTheDocument()
    })
```

Reply:
Ah I wasn't returning the `waitFor` 🤦